src/analyses/ai.h

```diff
@@ -124,6 +124,18 @@ class ai_baset
   {
   }
 
+  /// Output the abstract states for a single function
+  /// \param ns: The namespace
+  /// \param identifier: The identifier used to find a symbol to identify the
```

Comment:
`to identify the...` (copy and paste error?)

Reply:
Not really, the next parameter `goto_program,` is what is being identified here. But that's a bit weird to read. Fixed.